source/Octopus.Cli/Commands/ApiCommand.cs

```diff
@@ -66,15 +66,13 @@ protected ApiCommand(IOctopusClientFactory clientFactory, IOctopusAsyncRepositor
             options.Add("ignoreSslErrors", "[Optional] Set this flag if your Octopus Server uses HTTPS but the certificate is not trusted on this machine. Any certificate errors will be ignored. WARNING: this option may create a security vulnerability.", v => ignoreSslErrors = true);
             options.Add("enableServiceMessages", "[Optional] Enable TeamCity or Team Foundation Build service messages when logging.", v => commandOutputProvider.EnableServiceMessages());
             options.Add("timeout=", $"[Optional] Timeout in seconds for network operations. Default is {ApiConstants.DefaultClientRequestTimeout/1000}.", v => clientOptions.Timeout = TimeSpan.FromSeconds(int.Parse(v)));
-            options.Add("proxy=", $"[Optional] The URI of the proxy to use, eg http://example.com:8080.", v => clientOptions.Proxy = v);
+            options.Add("proxy=", $"[Optional] The URI of the proxy to use, e.g., 'http://example.com:8080'.", v => clientOptions.Proxy = v);
```

Comment:
```suggestion
            options.Add("proxy=", $"[Optional] The URI of the proxy to use, e.g., 'https://proxy.example.com:8080'.", v => clientOptions.Proxy = v);
```

_not sure if a proxy on port 808 would use https though???_

Reply:
Most popular squid example seems to be 443, so I've just removed the port. Also calling it "URL" instead of "URI" because it's more specific, less confusing.